sidebar/panel.js

```diff
@@ -531,11 +531,21 @@ function start() {
     },
 
     getFolderClass(state) {
+      return "folderEnabled";
+      /*
```

Comment:
No need for the comment, better if you just delete it.

I honestly can't remember why the folder icon has a different look based on download state 😕😄

Reply:
Done with the next commit. I would still leave a comment there, specially since we are not using the state parameter anymore.